### PR TITLE
Errata update for 4.6.8 release

### PR DESCRIPTION
**Pull request by codyhoag**
Ships 2020-12-14

### CODE REVIEW

**Comment by codyhoag** [comment]
@miabbott can you verify the 4.6.7 errata note for updated boot images?

https://github.com/openshift/openshift-docs/pull/27832/files#diff-3210eb8b77dfc42cf53d0a68f1a23845adfe8520940415e3c3d4abfc22f5ed68R2586-R2589

**Comment by codyhoag** [comment]
@sdodson can you verify the 4.6.7 errata note for the `eus-4.6` upgrade channel?

https://github.com/openshift/openshift-docs/pull/27832/files#diff-3210eb8b77dfc42cf53d0a68f1a23845adfe8520940415e3c3d4abfc22f5ed68R2591-R2594

**Comment by miabbott** [comment]
LGTM

**Comment by codyhoag** [comment]
Preview: https://rns-4-6-7--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-7

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2568,3 +2568,32 @@ To upgrade an existing {product-title} 4.6 cluster to this latest release, see x
...
+
+link:https://access.redhat.com/solutions/5625331[{product-title} 4.6.8 container image list]
+
+[id="ocp-4-6-8-features"]
+==== Features
+
+[id="ocp-4-6-8-new-rhcos-boot-images"]
+===== New {op-system-first} boot images available
+
+New {op-system} boot images are now available as part of the {product-title} 4.6.7 release. The updated {op-system} boot images provide improvements to the cluster boot experience. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899176[*BZ#1899176*])
```
Comment (miabbott) [inline release_notes/ocp-4-6-release-notes.adoc:2589]:
```suggestion
New {op-system} boot images are now available as part of the {product-title} 4.6.8 release. The updated {op-system} boot images provide improvements to the cluster boot experience. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899176[*BZ#1899176*])
```

Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc]: Good catch! Just updated; thanks!

**Pull request merged**